treescript/tests/test_util_task.py

```diff
@@ -1,9 +1,11 @@
+import copy
```

Comment:
This import is unused, could be removed (my bad, it's cruft I left around)

Reply:
Okay, fixed!